backend-zs/controllers/stories.controller.js

```diff
@@ -0,0 +1,14 @@
+var Story = require('../models/story.model').Story;
+
+exports.getStories = function (_req, res) {
+    Story.find()
+        .select("title author shortDescription numberLikes category")
+        .then(stories => { res.send(res.json(stories)) })
+        .catch(error => res.send(res.json({ error: error.message })))
+};
+
+exports.getStory = function (req, res) {
+    Story.findById(req.params.id)
+        .then(story => res.send(res.json(story)))
+        .catch(error => res.send(res.json({ error: error.message })))
```

Comment:
```suggestion
        .catch(error => res.status(500).json({ error: error.message }))
```
Returning a relevant status code for errors could be useful for the client using the API